index.html

```diff
@@ -346,7 +199,63 @@ <h2>Contact me</h2>
+              <p class="pp-dscrptn v-m">
+                Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent
+              </p>
+              <p class="pp-dscrptn v-d">
+                Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea
+              </p>
+              <div class="pp-flex">
+                <div class="tags pp-specs">
+                        <div class="tag-html pp-tag">
+                          <span>html</span>
+                        </div>
+                        <div class="tag-css pp-tag">
+                          <span>css</span>
+                        </div>
+                        <div class="tag-js pp-tag">
+                          <span>javaScript</span>
+                        </div>
+                        <div class="tag-html pp-tag v-d">
+                          <span>github</span>
+                        </div>
+                        <div class="tag-css pp-tag v-d">
+                          <span>ruby</span>
+                        </div>
+                        <div class="tag-js pp-tag v-d">
+                          <span>Bootstrap</span>
+                        </div>
+                </div>
+                <hr class="solid">
+                <div class="pp-btns">
+                  <div class="pp-btn">
+                    <a class="see" href="https://nedramrez.github.io/zema-portfolio">
+                        <div class="see-txt">See live</div>
+                        <img src="assets/images/blue-export.svg" alt="blue export button">
+                    </a>
+                    <a class="see" href="https://github.com/Nedramrez/portfolio">
+                        <div class="see-txt">See Source</div>
+                        <img src="assets/images/blue-github.svg" alt="blue-export-icon">
```

Comment:
- [ ] Kindly add a little more space between your technologies and link buttons in the mobile version so it conforms to the figma design.

> Yours
![image](https://user-images.githubusercontent.com/90556221/189330773-58192962-74dc-48f6-bbbf-b01259757c99.png)

> Figma
![image](https://user-images.githubusercontent.com/90556221/189330852-3f25af46-e2fd-4b77-b287-b953387bc9df.png)

- [ ] Kindly also remove the underline from your links.

- [ ] For the desktop version, please ensure that the overall styling conforms to the Figma template provided. Right now there is a great difference in between the designs of the two

> Yours
![image](https://user-images.githubusercontent.com/90556221/189331258-7e171939-f7ef-4833-9589-d1aca75067ec.png)

> Figma
![image](https://user-images.githubusercontent.com/90556221/189331417-756f7b28-ed39-4382-9819-75d8455f2c05.png)

You currently haven't added the sub-heading `Canopy`, the font size and color of your description are different and there is a difference in the container width so the content loses its proportionality in the desktop version. Kindly fix all of these issues